di_website/static/css/di_website.css

```diff
@@ -82,3 +82,7 @@
 .hovertext path {
   stroke: transparent !important;
 }
+
+.three-digit-footnote {
```

Comment:
Actually, add this padding to the `footnotes_item` class, that way they're all aligned
Instead of this:
![image](https://user-images.githubusercontent.com/5672438/92567378-45def300-f286-11ea-80b0-7a28df4cda81.png)

We get this:
![image](https://user-images.githubusercontent.com/5672438/92567445-61e29480-f286-11ea-92f6-230406ec131f.png)

It keeps them aligned all the way to 999, plus u don't need the template condition.